modules/lvms-about-lvm-storage-installation.adoc

```diff
@@ -20,6 +20,8 @@ When using {lvms} on multi-node clusters, {lvms} only supports provisioning loca
 
 The prerequisites to install {lvms} are as follows: 
 
+* Before installing {lvms}, ensure that you have a minimum of 10 milli CPU and 100Mi of RAM.
```

Comment:
not sure if "milli CPU" is the right way to put this. Other than that, lgtm :)

Reply:
Updated it to "millicpus" based on an example from the existing doc - https://docs.openshift.com/container-platform/4.7/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html

"millicpus" term is also used in kubernetes doc - https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes.